cmd/ethrex/cli.rs

```diff
@@ -51,6 +52,14 @@ pub struct Options {
         help_heading = "Node options"
     )]
     pub datadir: String,
+    #[arg(
+        long = "force", 
+        help = "Force remove the database",
+        long_help = "Delete the database without confirmation`.",
+        action = clap::ArgAction::SetTrue,
+        help_heading = "Node options"
+    )]
+    pub force: bool,
```

Comment:
What is this argument used for?

Reply:
I asked for it. Since maybe you want to execute a this command programmatically, then you don't want the prompt.

Reply:
But this argument is in the regular command, aside from the one already on the removedd subcommand

Reply:
Oh I didn't see we had removed both as a subcommand and as a flag, do we need both of them?

Reply:
In any case, I guess that is discussion for another PR/Issue